tests/cases/standalone/common/types/decimal/decimal_cast.result

```diff
@@ -292,92 +292,52 @@ SELECT 1.25::FLOAT::DECIMAL(3,2);
 -- overflow
 SELECT 100::FLOAT::DECIMAL(3,1);
 
-+------------+
-| Int64(100) |
-+------------+
-| 10.0       |
-+------------+
+Error: 3001(EngineExecuteQuery), Invalid argument error: 1000 is too large to store in a Decimal128 of precision 3. Max is 999
```

Comment:
Why were these cases filed? Do we need to investigate them?

Reply:
This behavior is correct

Reply:
Yes, i got it finally.